vendor/assets/javascripts/brainstem/brainstem-model.coffee

```diff
@@ -95,6 +105,16 @@ class window.Brainstem.Model extends Backbone.Model
     else
       super(field)
 
+  set: (key, val, options) ->
+    super
+
+    _.extend(@unsavedAttributes || {}, @changed)
```

Comment:
@juanca you're probably right, `initialize` makes more sense. I put it in the `constructor` to prevent sub-classes from having to call `super` when implementing `intitialize`... but how often are we sub-classing our models ¯\_( ͠° ͟ʖ °͠ )_/¯